src/dyn_model.jl

```diff
@@ -38,10 +41,17 @@ mutable struct DynamicPowerNetwork
     τ
     η_c
     η_d
+    ρ
+end
+
+function DynamicPowerNetwork(
+    fq, fl, pmax, gmax, A, B, P, C, T; 
+    τ=TAU, η_c=1.0, η_d=1.0, ρ=nothing
+)
+    ρ = something(ρ, 2*[maximum(x -> x[i], gmax) for i in 1:length(gmax[1])])
```

Comment:
Can you add one line explaining what the underlying assumption here is (I am understanding not more that `2gmax` per timestep?